### DISCUSSION TITLE
Copilot Chat extension in Visual Studio scrolls violently

### DISCUSSION THREAD

**Discussion by pavel-mikula-sonarsource** [community/community#120990] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Hi,

I've installed GitHub Copilot Chat for Visual Studio and the UX is causing me a headache. 

When the copilot is writing the answers, it violently jumps line by line in slightly irregular time intervals. This is very disruptive and impossible to read on the fly.

Please make the scrolling fluent in some way, and limit the amount of jumps.

Thank you

**Comment by dlabrecquemeca** [comment, marked as answer]
This seems to have been fixed in the update: https://developercommunity.visualstudio.com/t/GitHub-Copilot-Chat-window-auto-scrolls-/10950911 as of Version 17.14.14

**Comment by pavel-mikula-sonarsource** [comment]
The answer generation seems to be slower now, but it indeed doesn't jump so violently anymore
